cypress/integration/v3/blocks/blockquote/index.js

```diff
@@ -0,0 +1,47 @@
+
+/**
+ * External dependencies
+ */
+import {
+	assertBlockExist, blockErrorTest,
+} from '~stackable-e2e/helpers'
+
+export {
+	blockExist,
+	blockError,
+	typeContent,
+	pressEnterKey,
+}
+
+function blockExist() {
+	it( 'should show the block', assertBlockExist( 'stackable/blockquote', '.stk-block-blockquote' ) )
+}
+
+function blockError() {
+	it( 'should not trigger block error when refreshing the page', blockErrorTest( 'stackable/blockquote' ) )
+}
+
+function typeContent() {
+	it( 'should allow typing in the block', () => {
+		cy.setupWP()
+		cy.newPage()
+		cy.addBlock( 'stackable/blockquote' )
+		cy.typeBlock( 'stackable/text', '.stk-block-text__text', 'Lorem ipsum dolor sit amet.', 0 )
+			.assertBlockContent( '.stk-block-text__text', 'Lorem ipsum dolor sit amet.' )
+	} )
+}
+
+function pressEnterKey() {
+	it( 'should test clicking the enter key in text block', () => {
```

Comment:
```suggestion
	it( 'should test pressing the enter key in text block', () => {
```

Reply:
What does this test actually test?

Reply:
This is to test if pressing the enter key will show a block error. Applicable to Text block & Heading block since we expect them to add another block below them. (I guess not really necessary in other blocks such as here - blockquote ?)